### PR TITLE
pre-release docs updates

### PR DESCRIPTION
**Pull request by erohmensing**
**Proposed changes**:
- review a few more pages on https://github.com/RasaHQ/rasa/issues/6775
- remove references to the release candidate
- update sidebar for http api 
- auto-expand the main things in the sidebar (https://rasa-hq.slack.com/archives/C016FLYV4QN/p1601458540058800?thread_ts=1601455391.052100&cid=C016FLYV4QN) 
- hide "miragte from" until these pages are checked again (https://github.com/RasaHQ/rasa/issues/6474)

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Review by akelad — APPROVED** [review]

File: docs/docs/http-api.mdx [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
-title: Using the HTTP API
+sidebar_label: HTTP API
+title: Rasa Open Source HTTP API
 description: Read about Rasa's HTTP API that has endpoints for conversations, training models, and configuring your bot.
+abstract: You can use the HTTP API to interact with a running Rasa Open Source server. With the API, you can train models,
+  send messages, run tests, and more.
 ---
 
+:::tip Looking for API endpoints? 
+Check out the [API Spec](pages/http-api) for all of the available endpoints as well as their request and response formats.
```
Comment (m-vdb) [inline docs/docs/http-api.mdx:11]: @erohmensing @akelad the lint is failing here:
```suggestion
Check out the [API Spec](/pages/http-api) for all of the available endpoints as well as their request and response formats.
```

**Comment by tmbo** [comment]
@erohmensing can you take a look at the merge conflict?

**Comment by akelad** [comment]
fixed, do we force merge? :D

**Comment by tmbo** [comment]
sure, there is going to be enough builds to catch if something wen't wrong 😂

**Pull request merged**